catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ReliableResourceCallable.java

```diff
@@ -284,7 +305,26 @@ public ReliableResourceStatus call() {
                             bytesRead.get());
             reliableResourceStatus.setMessage("Download completed successfully");
         }
-
+        IOUtils.closeQuietly(cacheFileOutputStream);
         return reliableResourceStatus;
     }
+
+    private void closeAndDeleteCacheFile() {
+
```

Comment:
Is it okay to use `closeQuietly` and `deleteQuietly`?  If there are errors, we will not know.